core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -4502,7 +4516,7 @@ group by deptno]]>
     <Resource name="plan">
       <![CDATA[
 LogicalAggregate(group=[{0}], C=[AGGREGATE($1)])
-  LogicalProject(DEPTNO=[$7], COUNT_PLUS_100=[$9], COUNT_TIMES_100=[$10])
+  LogicalProject(DEPTNO=[$7], COUNT_PLUS_100=[$9])
```

Comment:
why is there a count?

Reply:
I might be misunderstanding your question but there is a count in the original query (see new line 4512: `select deptno, aggregate(count_plus_100) as c`)

Reply:
Thanks, I misread the diff file.

Reply:
No worries, appreciate you taking a look!